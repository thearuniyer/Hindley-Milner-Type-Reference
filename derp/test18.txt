{
	c = + < a b > x y;
}