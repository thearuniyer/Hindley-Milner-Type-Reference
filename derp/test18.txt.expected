c: bool #
a, b: ? #
x, y: ? #